pypit/arparse.py

```diff
@@ -1002,15 +1002,14 @@ def reduce_flatfield_method(self, v):
         v : str
           value of the keyword argument given by the name of this function
         """
-        allowed = ['polyscan', 'bspline']
+        allowed = ['bspline']
```

Comment:
My feeling is that the bspline does a better job, and there was a bug with the polyscan algorithm near the edges. So, if people did use it in the past, it's probably best that they don't anymore...